src/api.d.ts

```diff
@@ -186,6 +187,11 @@ export type ResolvedAppResourceTreeItem<T extends ResolvedAppResourceBase> = App
 
 export type LocalResource = AzExtTreeItem;
 
+export type ResolveResult<T> = {
```

Comment:
What does this get used for?

Reply:
It was unused, I removed it

Reply:
Ah, I see the comment in the other thread; leftovers from experimentation.